ext/http/01_http.js

```diff
@@ -378,6 +387,9 @@
     setEventTargetData(socket);
     socket[_server] = true;
     response[_ws] = socket;
+    if (options.idleTimeout !== 0) {
+      socket[_idleTimeoutDuration] = options.idleTimeout ?? 120;
+    }
```

Comment:
```suggestion
    socket[_idleTimeoutDuration] = options.idleTimeout ?? 120;
```
Conditionally setting properties causes property lookups to go polymorphic or megamorphic. Bad for performance.

Set `socket[_idleTimeoutTimeout]` to _something_ (e.g. undefined) for the same reason.